src/Messages/BodyFactories/Standard.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Puzzle\AMQP\Messages\BodyFactories;
+
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+use Puzzle\AMQP\Messages\TypedBodyFactories;
+use Puzzle\AMQP\Messages\BodyFactory;
+use Puzzle\AMQP\Messages\ContentType;
+use Puzzle\AMQP\Messages\Bodies\NullBody;
+use Puzzle\AMQP\Messages\TypedBodyFactory;
+
+class Standard implements BodyFactory
+{
+    use LoggerAwareTrait;
+    
+    private
+        $factories;
+    
+    public function __construct()
+    {
+        $this->logger = new NullLogger();
+        $this->initializeFactories();
+    }
+    
+    private function initializeFactories()
+    {
+        $this->factories = [
+            ContentType::TEXT => new TypedBodyFactories\Text(),
+            ContentType::JSON => new TypedBodyFactories\Json(),
+            ContentType::BINARY => new TypedBodyFactories\Binary(),
+        ];
+    }
+    
+    public function handleContentType($contentType, TypedBodyFactory $factory)
```

Comment:
Nope
BodyFactory is a contract for an object which can build Body instance from amqp message payload.
This method is to allow to customize the behaviour of this very implementation of BodyFactory